test/framework/modules.py

```diff
@@ -465,6 +466,23 @@ def test_module_caches(self):
         self.assertEqual(mod.MODULE_AVAIL_CACHE, {})
         self.assertEqual(mod.MODULE_SHOW_CACHE, {})
 
+    def test_module_use_bash(self):
+        """Test whether effect of 'module use' is preserved when a new bash session is started."""
+        # this test is here as check for a nasty bug in how the modules tool is deployed
+        # cfr. https://github.com/hpcugent/easybuild-framework/issues/1756,
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1326075
+        modules_dir = os.path.abspath(os.path.join(self.test_prefix, 'modules'))
+        self.assertFalse(modules_dir in os.environ['MODULEPATH'])
+
+        mkdir(modules_dir, parents=True)
+        self.modtool.use(modules_dir)
+        modulepath = os.environ['MODULEPATH']
+        self.assertTrue(modules_dir in modulepath)
+
+        out, _ = run_cmd("bash -c 'echo MODULEPATH: $MODULEPATH'", simple=False)
+        self.assertEqual(out.strip(), "MODULEPATH: %s" % modulepath)
+        self.assertTrue(modules_dir in out)
```

Comment:
strictly speaking, I think this is redundant, no? (I mean, how would you pass the above 2 and this fails).
I'm curious if you have a particular failure mode in mind or, it's just verbose because it's testing...

otherwise really good, ready for merging! that was fast!